lib/elixir/lib/calendar/datetime.ex

```diff
@@ -16,17 +16,21 @@ defmodule DateTime do
   and instead rely on the functions provided by this module as
   well as the ones in third-party calendar libraries.
 
-  ## Where are my functions?
+  ## Time zone database
 
-  You will notice this module only contains conversion
-  functions as well as functions that work on UTC. This
-  is because a proper `DateTime` implementation requires a
-  time zone database which currently is not provided as part
-  of Elixir.
+  Many functions in this module requires a time zone database.
+  By default, it uses the default time zone database returned by
+  `Calendar.get_time_zone_database/1`, which defaults to
+  `Calendar.UTCOnlyTimeZoneDatabase` which only handles "Etc/UTC"
+  datetimes and returns `{:error, :utc_only_time_zone_database}`
+  for any other time zone.
 
-  Such may be addressed in upcoming versions, meanwhile,
-  use third-party packages to provide `DateTime` building and
-  similar functionality with time zone backing.
+  Another time zone database, such as one originating from a package,
+  can be passed as argument or set globally, either via configuration:
+
+      config :elixir, :time_zone_database, CustomTimeZoneDatabase
+
+  or by calling `Calendar.put_time_zone_database/1`.
```

Comment:
Do we really want to duplicate the documentation text like that? I'm pretty sure they will get out of sync pretty quickly. I would forward from one to another - I'm not sure which way, though.

Reply:
Ok, that's fair. Should we add some comment around the docs that you should change the other place as well?